BUILDING.md

```diff
@@ -417,41 +417,50 @@ To build the documentation:
 
 This will build Node.js first (if necessary) and then use it to build the docs:
 
-```console
-$ make doc
+```bash
+make doc
 ```
 
 If you have an existing Node.js build, you can build just the docs with:
 
-```console
-$ NODE=/path/to/node make doc-only
+```bash
+NODE=/path/to/node make doc-only
 ```
 
-To read the documentation:
+To read the man page:
 
-```console
-$ man doc/node.1
+```bash
+man doc/node.1
 ```
 
-If you prefer to read the documentation in a browser,
-run the following after `make doc` is finished:
+If you prefer to read the full documentation in a browser, run the following
+after `make doc` is finished:
 
-```console
-$ make docopen
+```bash
+make docserve
 ```
 
-This will open a browser with the documentation.
+This will provide a URL to where you may browse the documentation locally.
+
+If you prefer to read the documentation in a text-based browser, run the
+following after `make doc` is finished:
+
+```bash
+make docopen
+```
+
+This will open a text-based browser with the documentation in your terminal.
```

Comment:
For me, `make docopen` opens my default system browser, not a text-based browser and not in my terminal. (I'm on macOS.) Probably best to leave the previous wording?

Reply:
Okay, will do.